openstack/baremetal/v1/nodes/testing/fixtures.go

```diff
@@ -631,6 +717,14 @@ const NodeSingleBIOSSettingBody = `
 }
 `
 
+func getIntPointer(x int) *int {
+	return &x
+}
+
+func getBoolPointer(x bool) *bool {
+	return &x
+}
```

Comment:
There are a few existing ways of handling this:

1. There's some existing helper code here: openstack/identity/v3/users/testing/requests_test.go
2. Just declare the variables each time: https://github.com/gophercloud/gophercloud/blob/513734676e6495f6fec60e7aaf1f86f1ce807428/openstack/networking/v2/extensions/mtu/testing/results_test.go#L90-L94

I can't recall why there aren't central helper functions in the `testhelper` package. I have a feeling that, for whatever reason, it was discouraged to do this in Golang in general and to instead manually declare every occurrence of pointers, but it's been too long for me to remember at this point.

Reply:
Thank you. I changed to use the variables.